crates/core/src/db/db_metrics/mod.rs

```diff
@@ -129,6 +129,11 @@ metrics_group!(
         #[help = "For a given module, the size of its log file (in bytes)"]
         #[labels(db: Address)]
         pub module_log_file_size: IntGaugeVec,
+
+        #[name = spacetime_request_round_time]
+        #[help = "The total time it takes for request to complete"]
+        #[labels(txn_type: WorkloadType, database_address: Address, reducer_symbol: str)]
+        pub request_round_trip: HistogramVec,
```

Comment:
@joshua-spacetime , I am not very sure if this metric should be a part of db_metrics or worker_metrics.

Reply:
I don't think it matters, but maybe worker_metrics.